lib/axon/losses.ex

```diff
@@ -1134,6 +1123,14 @@ defmodule Axon.Losses do
     t0_prob
   end
 
+  deftransformp ctc_reduction(loss, l_true, reduction) do
+    case {reduction, loss} do
+      {:mean, loss} -> Nx.divide(loss, l_true) |> Nx.mean()
+      {:sum, loss} -> Nx.sum(loss)
+      {:none, loss} -> loss
+    end
```

Comment:
This might just work in `defn` too:

    case reduction do
      :mean -> Nx.divide(loss, l_true) |> Nx.mean()
      :sum -> Nx.sum(loss)
      :none -> loss
    end

Reply:
To be clear, `case atom do` should work on `defn`. :)